js/widgets/widget-stickyHeaders.js

```diff
@@ -206,6 +206,18 @@
 							.addClass( state )
 							.css(cssSettings);
 					}
+					if (isVisible !== laststate) {
+							// make sure the focused filter stay focused after showing/hiding sticky
+							var $td = $(document.activeElement).closest('td'),
+							column = $td.parent().children().index($td);
```

Comment:
Hi! Thanks for contributing and I'm sorry for taking so long to follow up on this PR!

Would you please change this to get the cell's index from the `data-column` attribute since the header may have a `colspan` which would mess up this calculation.